h2o-core/src/main/java/hex/Model.java

```diff
@@ -1011,12 +1024,19 @@ public Model(Key<M> selfKey, P parms, O output) {
     super(selfKey);
     assert parms != null;
     _parms = parms;
+    evalAutoParamsEnabled = evaluateAutoModelParameters();
```

Comment:
if your goal is to make a small optimization by avoiding multiple evaluation of `Boolean.parseBoolean`, I would make `evalAutoParamsEnabled` an instance param, not a class param.

Having constructor modiying a static field is a pretty ugly and confusing pattern that should be avoided, especially if this value is then read by all instances (the only exception would be counters maybe, and even, there are cleaner way to do it).

Please change:
```   public static boolean evalAutoParamsEnabled = true; ```
into a private instance field
```    private boolean evalAutoParamsEnabled = true; ```

Reply:
hm yes, I agree, thx